[gpio] Update README.md (#9899)
openhab/openhab-addons: bundles/org.openhab.binding.gpio/README.md

-> Interfacing Options --> Remote GPIO --> YES --> OK --> Finish

> Note: if you are setting this up on a Raspberry Pi running openHabian, raspi-config is no longer available. However, you can still create the service config file manually:

Reviewer: I would suggest to reword it a bit different. Because if openHabian would get raspi-config in the future this comment will be outdated and thus would need to be changed. So I've generalized it to systems without raspi-config.:
```suggestion
Note: if you are setting this up on a Raspberry Pi without `raspi-config` you can create the service config file manually:
```

```
sudo mkdir -p /etc/systemd/system/pigpiod.service.d/
sudo nano /etc/systemd/system/pigpiod.service.d/public.conf
```
      [Service]
      ExecStart=

>       ExecStart=/usr/bin/pigpiod

Reviewer 2: Why are there two `ExecStart`s?

Author: Because this is what the raspi-config script adds to the pigpiod config file and I copied verbatim (in case it matters).
https://github.com/RPi-Distro/raspi-config/blob/74885481d8f311ab3875b6601153132552b41d67/raspi-config#L1616

Reviewer 2: It seems an error to me. Can you test if it still works if you remove the empty `ExecStart=`?

Reviewer: It seems to be to override the default values. See https://bugzilla.redhat.com/show_bug.cgi?id=756787. So I guess it is to be sure no default settings are present. That would mean even if @SebBoulet would test this on his system and it works, it might not work for other people who have other configurations. So I would say this is legitimate use case.

Reviewer 2: I didn't find that info when doing a quick search. Thanks!

```
sudo systemctl daemon-reload
```
Now that Remote GPIO is enabled, get the daemon going:
```
sudo systemctl enable pigpiod 
sudo systemctl start pigpiod
```

> In openHab, set `host` to the address of the pi and the `port` to the port of pigpio (default: 8888).

Reviewer: ```suggestion
In openHAB, set `host` to the address of the pi and the `port` to the port of pigpio (default: 8888).
```
